plugins/RssWidget/RssRenderer.php

```diff
@@ -56,7 +56,21 @@ public function get()
         if (!$output) {
             try {
                 $content = Http::fetchRemoteFile($this->url);
+
+                $promoteWarningToException = function ($errno, $errstr, $errfile, $errline, $errcontext) {
+                    // if the error has been suppressed by the @ we don't handle the error
+                    if (error_reporting() == 0) {
+                        return false;
+                    }
+                    if ($errno !== E_WARNING) {
+                        return false;
+                    }
+                    throw new \Exception($errstr, $errno);
+                };
+                set_error_handler($promoteWarningToException, E_WARNING);
```

Comment:
this would be overwriting our error handler? 

Can we maybe just do something like `$rss = @simplexml_load_string($content)`
and trigger an exception if $rss is empty?

Reply:
I restored it right afterwards with `restore_error_handler()`. But yeah, I guess just supressing it would be simpler. I'll do that.